src/main/java/co/cask/gcs/sink/GCSTextBatchSink.java

```diff
@@ -88,27 +77,37 @@ public void transform(StructuredRecord input,
     }
```

Comment:
not related to your change, but not sure if its desirable to silently consume the Exception.

Reply:
I agree, removing this.